src/components/Nav.tsx

```diff
@@ -28,7 +28,9 @@ const Nav = () => {
 			if (!data) {
 				dispatch(fetchUserProfile()).unwrap();
 			}
-			dispatch(getCarts()).unwrap();
+			if (userData?.role === 'Buyer') {
```

Comment:
okay i test the functionality of this product and we still have this issue on product page when is an admin logged in 
![Uploading Screenshot 2024-07-31 201457.png…]()